
syntax = "proto3";

package protobuf_unittest;

import "google/protobuf/any.proto";

message TestAny {
  int32 int32_value = 1;
  google.protobuf.Any any_value = 2;
  repeated google.protobuf.Any repeated_any_value = 3;
}
